admin/environment.yml

```diff
@@ -0,0 +1,21 @@
+name: pydata
+dependencies:
+- openssl=1.0.2h=1
+- pip=8.1.2=py35_0
+- python=3.5.2=0
```

Comment:
Probably should specify only the python version. The rest gets installed automatically I think.